src/main/java/com/gotech/accesscontrol/converter/UserConverter.java

```diff
@@ -0,0 +1,19 @@
+package com.gotech.accesscontrol.converter;
```

Comment:
create util package write CommonUtil and move this method in that class toUser(UserRequest userRequest)